src/routes/dashboard/ticket/+page.server.ts

```diff
@@ -5,7 +5,7 @@ import {
     getLabelsWithoutDeadline,
     getUserFromSession,
 } from '$lib/server/database';
-import { error, fail, redirect } from '@sveltejs/kit';
+import { error, redirect } from '@sveltejs/kit';
```

Comment:
This is a needed import.
```suggestion
import { error, fail, redirect } from '@sveltejs/kit';
```